src/index.html

```diff
@@ -7,12 +7,65 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <title>Stars</title>
+
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./page.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./stars.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <main>
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars stars--1">
+        <div class="stars__star active"></div>
```

Comment:
```suggestion
        <div class="stars__star"></div>
```
rewrite the task without additional classes